e2e/fixtures/broken-links/src/pages/_layout.tsx

```diff
@@ -5,12 +5,7 @@ export default async function RootLayout({
 }: {
   children: ReactNode;
 }) {
-  return (
-    <html>
-      <head></head>
-      <body>{children}</body>
-    </html>
-  );
+  return children;
```

Comment:
Maybe we can remove the file entirely.

Reply:
removed. Do you think the other layouts that just return children should also be removed or do we want to keep some of them?

Reply:
Let's remove.

Reply:
removed 👍